bot/bot.go

```diff
@@ -837,6 +821,36 @@ func flipCoin(ctx context.Context, b *Bot, event *slack.MessageEvent) {
 	}
 }
 
+// postToPlayground converts the text of a post into code for the playground. It is not perfect but works most of the time.
+// Text outside of ``` quotes is converted into a comment and included in the code, everything inside of those quotes is
+// considered code and pasted as-is.
+func postToPlayground(text string) string {
+	var pg string
+	parts := strings.Split(text, "```")
```

Comment:
Should we just bail if this is `< 2`?

Reply:
Not sure, I've seen a few posts where the author forgot the second ```.

Reply:
True, good point. 👍